contracts/OffsetHelper.sol

```diff
@@ -287,13 +287,15 @@ contract OffsetHelper is OffsetHelperStorage {
         while (remainingAmount > 0 && i < scoredTCO2Len) {
             address tco2 = scoredTCO2s[i];
             uint256 balance = IERC20(tco2).balanceOf(address(this));
-            i += 1;
-            if (balance == 0) continue;
+            if (balance < 1e15) continue;
```

Comment:
What is this check doing? What is "1e15"?

Reply:
This checks whether `balance` is less than `1e15` which is a concise way to write 1000000000000000, or 0.001 TCO2 or 1 kilo of co2.

https://stattrek.com/statistics/dictionary.aspx?definition=E-Notation

Reply:
Oh, it's less than a kilo amounts. Got it.